### PR TITLE
sidfj

### PR DESCRIPTION
**Pull request by josi3**
dsa

### CODE REVIEW

**Comment by mk-boop** [comment]
jkjlkjl

**Comment by mk-boop** [comment]
huihui

File: etc/ok [hunk 1]
```diff
@@ -1,5 +1 @@
-ok
-ok2
-ok3
```
Comment (mk-boop) [inline etc/ok:3]: Those must stay

File: src/Kernel.php [hunk 2]
```diff
@@ -1,5 +1,7 @@
 <?php
-
+/**
+ * ok
```
Comment (mk-boop) [inline src/Kernel.php:3]: Fine

File: etc/ok (hunk 1 above)
Comment (josi3) [reply etc/ok:3]: No way

File: src/Kernel.php (hunk 2 above)
Comment (josi3) [reply src/Kernel.php:3]: Ok

File: etc/ok (hunk 1 above)
Comment (josi3) [reply etc/ok:3]: @josi3 u mad?

Comment (mk-boop) [reply etc/ok:3]: @josi3 u mad?

Comment (josi3) [reply etc/ok:3]: nah ok

**Comment by mk-boop** [comment]
gvjhbhbkj

**Pull request merged**